tracing-subscriber/tests/ansi_fmt.rs

```diff
@@ -0,0 +1,132 @@
+mod support;
+use self::support::*;
+use lazy_static::lazy_static;
+use std::io::Cursor;
+use tracing::{self, subscriber::with_default, Level};
+use tracing_subscriber::{filter::EnvFilter, fmt::time::FormatTime, fmt::MakeWriter, prelude::*};
+
```

Comment:
What do you think about moving these from the integration tests directory into a `tests` module in the `fmt` module? That way, the `MockWriter` could be factored out in a `#[cfg(test)]` test-support module somewhere, and used by both these tests _and_ the `writer.rs` tests.

Reply:
Good idea, see new commits